include/lsst/afw/geom/SeparableXYTransform.h

```diff
@@ -41,6 +41,11 @@ class Functor;
 /** @brief A 2D transform for which the pixel distortions in the in
  *  the x- and y-directions are separable.
  *
+ * @deprecated Deprecated in 14.0. Will be removed in 15.0. It is partially
+ *             replaced by ast::ParallelMap or ast::Mapping::under, but the
+ *             replacement of Functors with Mappings must be done on a
+ *             case-by-case basis.
+ *
```

Comment:
I suppose I just don't consider the cycle releases to be terribly meaningful from a code/interface content standpoint right now; the only release unit that's actually meaningful to developers are the weeklies.  (I'd say that weeklies are also the only unit that most external users should pay attention to right now, though that's not universal).  While we ought to have a release cadence in-between weekly and semiannually that actually gets backports so it can stay genuinely usable for more than a couple of weeks, we simply don't have that at present.